lib/we_got_coders/content.rb

```diff
@@ -0,0 +1,29 @@
+    last_name:"RUDOLPH",
+    learning_experience:"For me, a positive learning and training environment is highly personal, interactive and very hands-on.  Small classes with a mix of teamwork and independent learning is great.  I don’t mind a bit of pressure and my nature is such that I tend eat, sleep and breathe any project I’m working on.  I definitely need to learn by doing.  I’m far more likely to learn something if I’m working on a real-world problem than if it’s theoretical.  This is what prompted me to steer my degree from pure maths to applied maths.Throughout my life, I’ve always sought teams to co-learn with and collaboration is a core value of mine. I love the thrill of hashing out a problem with small teams and I've noticed I tend to learn from friends and make friends with those who I'm learning with.  For self-learning, I prefer a quiet environment, though am able to get absorbed in a problem no matter where I am!",
+    phone_number:"07471286289",
+    primes_sum:"",
+    projects:"I lead a quant project for an energy trading desk which involved designing an entire risk system from scratch.  This was a 12 week project which involved various streams of work, including designing computational methodology, data sourcing (and back-up) and automation. I worked with a python programmer, though I coded most of the project in Matlab.  Our algorithms would daily source and pre-process a large amount of energy pricing data and then compute a range of risk metrics which were relied upon (and audited) by my employer, a global trading firm.",
+    rationale:"Having researched many bootcamps, We Got Coders is uniquely appealing and I would love to join your May cohort -  you are by far my  number one choice.  I’m really impressed by the choices you’ve made in designing your bootcamp.  I believe a small, intense and dedicated class is far preferable and gives me comfort that you’re investing in us to be excellent (not just churn out a lot of “ok” coders).    It demonstrates the respect that you have for your coding students and this is evident in the testimonials I’ve read. I’m also impressed that We Got Coders is a digital agency (not just an education initiative) because that means you are sourcing what you’re teaching according to relevance in the real-world.  I’m very keen to be learning Ruby, not only because it appears to be increasingly used by many companies, but because I've found it intuitive to work with and seems to have a great community. I’m also very keen on the idea of a live-in bootcamp because not only do I hate commuting, but I also think it’d be great to get to know everyone outside of the classroom in a more informal setting like evening meals etc. The beautiful countryside setting would definitely help me focus for the long days of coding and it’s another example of the great choices I think you’ve made.",
+    ruby_track_completedness:"",
+    twitter:"",
+    web_track_completedness:""
+     }
+
+end
```

Comment:
Hey Dan, thanks for the feedback!
So, yes,this was the key thing that caused me the most grief in the
challenge - I just couldn't work out why it didn't come down from the
server.  I looked directly here:
http://wegotcoders.com/trainees/profile?access_token=3cf5e6fd6f454967a6762af88e1ea90f8b955e1f7ec0b7f95ee8f79bd5512ef4
but those fields were empty.

And even after the /push when I had another look at response.body (which i
noticed had far more fields than the original @profile), there was still no
content for anything other than the short answer fields.  Likewise I
couldn't write to those fields either.  I tried different types of inputs
in case it was due to some kind of validation on the field. I also looked
at where you took the input for those fields and thought they may have been
hidden in some kind of other sub variable
trainee[applications_attributes][0][about].  But no luck and in the end I
gave up.  I'm relieved to hear that it shouldn't have been that hard!

But please let me know if it was something I was missing as I'd love to
still get it working :)

Cheers,
Tamlyn

*Tamlyn Rudolph*

T:    0439 544 034
E:   tamlyn@ethicalswitch.com
W:  ethicalswitch.com




On Mon, Mar 7, 2016 at 2:58 PM, Dan Garland <notifications@github.com>
wrote:

> In lib/we_got_coders/content.rb
> <https://github.com/wegotcoders/wgc_groundwork/pull/50#discussion_r55214055>
> :
>
> > +    github:"tamlyn10",
> > +    homepage:"",
> > +    how_did_you_hear:"Course Report!",
> > +    js_track_completedness:"",
> > +    last_name:"RUDOLPH",
> > +    learning_experience:"For me, a positive learning and training environment is highly personal, interactive and very hands-on.  Small classes with a mix of teamwork and independent learning is great.  I don’t mind a bit of pressure and my nature is such that I tend eat, sleep and breathe any project I’m working on.  I definitely need to learn by doing.  I’m far more likely to learn something if I’m working on a real-world problem than if it’s theoretical.  This is what prompted me to steer my degree from pure maths to applied maths.Throughout my life, I’ve always sought teams to co-learn with and collaboration is a core value of mine. I love the thrill of hashing out a problem with small teams and I've noticed I tend to learn from friends and make friends with those who I'm learning with.  For self-learning, I prefer a quiet environment, though am able to get absorbed in a problem no matter where I am!",
> > +    phone_number:"07471286289",
> > +    primes_sum:"",
> > +    projects:"I lead a quant project for an energy trading desk which involved designing an entire risk system from scratch.  This was a 12 week project which involved various streams of work, including designing computational methodology, data sourcing (and back-up) and automation. I worked with a python programmer, though I coded most of the project in Matlab.  Our algorithms would daily source and pre-process a large amount of energy pricing data and then compute a range of risk metrics which were relied upon (and audited) by my employer, a global trading firm.",
> > +    rationale:"Having researched many bootcamps, We Got Coders is uniquely appealing and I would love to join your May cohort -  you are by far my  number one choice.  I’m really impressed by the choices you’ve made in designing your bootcamp.  I believe a small, intense and dedicated class is far preferable and gives me comfort that you’re investing in us to be excellent (not just churn out a lot of “ok” coders).    It demonstrates the respect that you have for your coding students and this is evident in the testimonials I’ve read. I’m also impressed that We Got Coders is a digital agency (not just an education initiative) because that means you are sourcing what you’re teaching according to relevance in the real-world.  I’m very keen to be learning Ruby, not only because it appears to be increasingly used by many companies, but because I've found it intuitive to work with and seems to have a great community. I’m also very keen on the idea of a live-in bootcamp because not only do I hate commuting, but I also think it’d be great to get to know everyone outside of the classroom in a more informal setting like evening meals etc. The beautiful countryside setting would definitely help me focus for the long days of coding and it’s another example of the great choices I think you’ve made.",
> > +    ruby_track_completedness:"",
> > +    twitter:"",
> > +    web_track_completedness:""
> > +     }
> > +
> > +end
>
> Wasn't sure why you needed this hard-coded content as well. It *ought* to
> come down from the sever in the JSON and therefore be dynamic like the
> @profile part
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wegotcoders/wgc_groundwork/pull/50/files#r55214055>.
>